### PR TITLE
Add backgroundColor and border options to writeTextToCanvas background

### PR DESCRIPTION
**Pull request by tfili**
There has been multiple requests for this. Its not currently used in labels, but this can be used to create a canvas and add it as a billboard.

Tests were added and CHANGES.md was updated.

### CODE REVIEW

**Comment by pjcozzi** [comment]
LGTM.  @mramato do you want to test and merge?

**Comment by tfili** [comment]
@mramato I've updated the parameter name like we discussed.

File: CHANGES.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ Change Log
     * `Viewer.terrainShadows` now uses the `ShadowMode` enum instead of a Boolean, e.g. `viewer.terrainShadows = ShadowMode.RECEIVE_ONLY`.
 * Removed the default gamma correction for Bing Maps aerial imagery, because it is no longer an improvement to current versions of the tiles.  To restore the previous look, set the `defaultGamma` property of your `BingMapsImageryProvider` instance to 1.3.
 * Fixed billboard rotation when sized in meters. [#3979](https://github.com/AnalyticalGraphicsInc/cesium/issues/3979)
+* Added `DebugCameraPrimitive` to visualize the view frustum of a camera.
```
Comment (mramato) [inline CHANGES.md]: Not sure why these changes are in this branch.  Perhaps merge in master and make sure only your new entry is here.

**Comment by mramato** [comment]
That's my only comment.

**Comment by tfili** [comment]
@mramato Not sure what happened there. Updated changes.md
